pic-create

```diff
@@ -24,7 +24,7 @@ this_dir=`dirname $0`
 #only copy submit and params if we clone from default pic folder
 default_folder_to_copy="submit include/simulation_defines/param include/simulation_defines/unitless"
 #if we clone a project we copy full include 
-folder_to_clone="submit include"
+folder_to_clone="submit include lib/python"
```

Comment:
`lib/python` must be `lib` only. Rsync is going recursive throw all folder in lib